gradle/baseline.gradle

```diff
@@ -17,9 +17,25 @@ List<String> blacklistedBaselineProjects = [
         'profile-client-protobufs',
         'timestamp-impl']
 
+List<String> blacklistedErrorProneProjects = [
+        'atlasdb-console',
+        'atlasdb-dbkvs-hikari',
+        'atlasdb-jdbc',
+        'atlasdb-jdbc-tests',
+        'atlasdb-rocksdb',
+        'atlasdb-rocksdb-tests',
+        'atlasdb-service-server',
+        'cassandra-partitioner',
+        'commons-db',
+        'commons-executors',
+        'leader-election-impl',
+        'lock-impl',
+        'profile-client-protobufs',
+        'timestamp-impl']
 
 if (!blacklistedBaselineProjects.contains(project.name)) {
     apply plugin: 'com.palantir.baseline-checkstyle'
+} else if (!blacklistedErrorProneProjects.contains(project.name)) {
```

Comment:
This is more expicit. Thanks for changing this, but would like to see that `blacklistedErrorProneProjects` do not exist soon :)